db/schema.rb

```diff
@@ -43,6 +43,7 @@
     t.integer  "customer_id"
     t.integer  "c2_status",       default: 0,    null: false
     t.string   "token"
+    t.datetime "archived_at"
```

Comment:
Yeah we've had this issue on a few other things (he also had timestamps in copy for things like when an auction was published, which we do not record) and after talking to him I realized that he was not as concerned about the timestamp. 

We can keep if he thinks it is important, but each new db field is another thing to maintain so it would be really awesome to not add this if we dont need it